src/GuiBase/Viewer/Viewer.hpp

```diff
@@ -73,7 +73,12 @@ class RA_GUIBASE_API Viewer : public WindowQt, public KeyMappingManageable<Viewe
     QOpenGLContext* getContext() const { return m_context.get(); }
 
     /// Access to camera interface.
-    CameraInterface* getCameraInterface();
+    CameraManipulator* getCameraManipulator();
+
+    /// Set the current camera interface.
+    /// @note for compatibility, this method called without parameter sets the cameraInterface to
+    /// TrackballCamera
```

Comment:
hm.. do you mean when calling `setCameraManipulator( nullptr )` ?
I do not see the code doing what you state